### PR TITLE
Create excludeservice flag

### PR DESCRIPTION
**Pull request by chrisng93**
Create excludeservice flag. When set to true, thrift2flow skips flow/js generation for thrift service definitions.

### CODE REVIEW

File: src/main/convert.js [hunk 1]
```diff
@@ -58,12 +58,15 @@ export class ThriftFileConverter {
   constructor(
     thriftPath: string,
     transformName: string => string,
-    withsource: boolean
+    withsource: boolean,
+    excludeservice: boolean,
```
Comment (johnmanong) [inline src/main/convert.js:62]: [nit] naming should be pascalCase

File: src/main/index.js [hunk 2]
```diff
@@ -50,6 +50,15 @@ const argv = yargs
     describe: 'prepend the source path of the thrift file',
     default: false,
   })
+  // .option('includetypes', {
```
Comment (johnmanong) [inline src/main/index.js]: Please RM comment out code.

File: src/main/convert.js (hunk 1 above)
Comment (chrisng93) [reply src/main/convert.js:62]: yeah i would like pascalCase as well, but i'm just following the flag standards that uber has set for this package

File: src/main/index.js (hunk 2 above)
Comment (chrisng93) [reply src/main/index.js]: oops. good catch

File: src/main/convert.js (hunk 1 above)
Comment (johnmanong) [reply src/main/convert.js:62]: Sure. kinda confusing with the casing in args above, but agree that consistency is better here.

**Review by johnmanong — APPROVED** [review]

**Pull request merged**